rosa_cluster_admin/rosa-cluster-autoscaling.adoc

```diff
@@ -25,14 +38,19 @@ include::modules/rosa-cluster-autoscaler-ui-settings.adoc[leveloffset=+1]
 
 include::modules/rosa-cluster-autoscaler-cli-interactive-during.adoc[leveloffset=+1]
 
-include::modules/rosa-cluster-autoscaler-cli-interactive-after.adoc[leveloffset=+2]
+include::modules/rosa-cluster-autoscaler-cli-interactive-after.adoc[leveloffset=+1]
 
 include::modules/rosa-cluster-autoscaler-cli-during.adoc[leveloffset=+1]
 
-include::modules/rosa-cluster-autoscaler-cli-after.adoc[leveloffset=+2]
+include::modules/rosa-cluster-autoscaler-cli-after.adoc[leveloffset=+1]
+endif::openshift-rosa[]
+
+include::modules/rosa-cluster-autoscaler-cli-edit.adoc[leveloffset=+1]
 
-include::modules/rosa-cluster-autoscaler-cli-edit.adoc[leveloffset=+2]
+include::modules/rosa-cluster-autoscaler-cli-describe.adoc[leveloffset=+1]
```

Comment:
I might be missing something here, but this module is included at leveloffset=+1, but in the preview, it looks like leveloffset=+2 (i.e. "View autoscaler configurations with the ROSA CLI" is a child of "Edit autoscaling after cluster creation with the ROSA CLI" rather than a sibling).

Reply:
I accidentally left in an extra `=`. Thank you!